devops/tasks.py

```diff
@@ -309,79 +317,124 @@ def get_master_key(env: str) -> None:
     )
 
     content = result.stdout.decode(encoding="utf-8")
-    output_file = master_key_path(env=env)
 
-    logger.info(f"Saving master key to {output_file}")
+    logger.info(f"Saving master key to {master_key_file}")
 
-    output_file.write_text(content, encoding="utf-8")
+    master_key_file.write_text(content, encoding="utf-8")
+    return master_key_file
 
 
-def unseal_secrets(env: str) -> None:
+def unseal_secrets(envs: Iterable[str]) -> None:
```

Comment:
Wouldn't it be easier and clearer if this iteration was done outside of this function in the few places it's needed in?

Reply:
True, changed that.